server/controllers/finance/fiscalPeriod.js

```diff
@@ -103,3 +105,12 @@ function getPeriodDiff(periodIdA, periodIdB) {
   return db.one(sql, [periodIdA, periodIdB]);
 }
 
+function currentPeriod(req, res, next) {
+  const sql = `SELECT id FROM period WHERE start_date <=? AND end_date >=?`;
+  const currentDate = util.dateFormatter(new Date());
```

Comment:
It's probably easier to let SQL format these.  Like this:
```js
const sql = `
  SELECT id FROM period WHERE DATE(start_date) <= DATE(?) AND DATE(end_date) >= DATE(?);
`;

db.one(sql, [new Date(), new Date()])
  .then(period => res.status(200).json(period.id));
  .catch(next)
  .done();
```

[According to the documentation for mysqljs](https://github.com/mysqljs/mysql#escaping-query-values)
> Date objects are converted to `'YYYY-mm-dd HH:ii:ss'` strings

This is a valid [DATETIME representation](https://dev.mysql.com/doc/refman/5.7/en/datetime.html) in MySQL.  However, since we don't care about the hour/minute/second parts, passing them to the SQL DATE() function will remove those parts, leaving just the DATE part for comparison.